src/socket.js

```diff
@@ -1,9 +1,13 @@
 import action from '@/store/action-types';
 
 const socket = new WebSocket(`ws://${window.location.host}/ws/`);
+const messageQueue = [];
```

Comment:
Change name to emphasize the fact that this is only used before the connection is made